tests/integration-tests/tests/storage/test_ephemeral/test_head_node_stop/pcluster.config.yaml

```diff
@@ -0,0 +1,31 @@
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: {{ instance }}
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+  Imds:
+    Secured: {{ imds_secured }}
+  LocalStorage:
+    EphemeralVolume:
+      MountDir: {{ head_ephemeral_mount }}
+  Iam:
+    AdditionalIamPolicies:
+      - Policy: arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore
+Scheduling:
+  Scheduler: {{ scheduler }}
+  {% if scheduler == "awsbatch" %}AwsBatchQueues:{% else %}SlurmQueues:{% endif %}
```

Comment:
If we only define the "awsbatch" as scheduler in test config, we don't need this jinja condition in config file.

Reply:
I prefer in having this choice so test configuration can be changed to use "slurm" at no cost